src/citrine/resources/generative_design_execution.py

```diff
@@ -0,0 +1,112 @@
+"""Resources that represent both individual and collections of design workflow executions."""
+import sys
+from typing import Optional, Union, Iterator
+from uuid import UUID
+
+from citrine._rest.collection import Collection
+from citrine._utils.functions import shadow_classes_in_module
+from citrine._session import Session
+import citrine.informatics.executions.design_execution
+from citrine.informatics.executions.generative_design_execution import GenerativeDesignExecution
+from citrine.informatics.scores import Score
+from citrine.resources.response import Response
+
+
+shadow_classes_in_module(citrine.informatics.executions.design_execution, sys.modules[__name__])
+
+
+class GenerativeDesignExecutionCollection(Collection["GenerativeDesignExecution"]):
+    """A collection of GenerativeDesignExecutions."""
+
+    _path_template = '/projects/{project_id}/generative-design/executions'
+    _individual_key = None
+    _collection_key = 'response'
+    _resource = GenerativeDesignExecution
+
+    def __init__(self, project_id: UUID, session: Session):
+        self.project_id: UUID = project_id
+        self.session: Session = session
+
+    def build(self, data: dict) -> GenerativeDesignExecution:
+        """Build an individual GenerativeDesignExecution."""
+        execution = GenerativeDesignExecution.build(data)  # this is inherited [Serialization]
+        execution._session = self.session
+        execution.project_id = self.project_id
+        return execution
+
+    # TODO: The execution_input should be a DesignExecutionInput(?), not a Score
```

Comment:
Yes, you'll need to create a Python class that holds the settings used to kick off the generative execution